WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItemBuilderParams.kt

```diff
@@ -66,7 +67,8 @@ sealed class MySiteCardAndItemBuilderParams {
         val bloggingPromptCardBuilderParams: BloggingPromptCardBuilderParams,
         val promoteWithBlazeCardBuilderParams: PromoteWithBlazeCardBuilderParams,
         val dashboardCardDomainBuilderParams: DashboardCardDomainBuilderParams,
-        val pagesCardBuilderParams: PagesCardBuilderParams
+        val pagesCardBuilderParams: PagesCardBuilderParams,
+        val activityBuilderParams: ActivityCardBuilderParams
```

Comment:
np: `activityBuilderParams` is to me neither consistent with the other param names nor "context-aware". I would either make it consistent if I were someone else, or make it context-aware if I were coding this 😄 

- consistent: `activityCardBuilderParams`
- context-aware: `activityParams`… we're in a builder already 😄

Reply:
🤦 Looks like I forgot to add the word "Card" - I can't tell you how many times I read that and didn't see the missing word. Thanks for picking it up.

Reply:
> context-aware: activityParams… we're in a builder already
You mean go against all the other "params" names and drop Builder? Oh my! This IS a great topic for the tech chat!